vendor/knative.dev/pkg/test/helpers/name.go

```diff
@@ -48,11 +48,7 @@ func ObjectPrefixForTest(t test.T) string {
 
 // ObjectNameForTest generates a random object name based on the test name.
 func ObjectNameForTest(t test.T) string {
-	prefix := ObjectPrefixForTest(t)
-	if len(prefix) > 20 {
-		prefix = prefix[:20]
-	}
-	return kmeta.ChildName(prefix, string(sep)+RandomString())
+	return kmeta.ChildName(ObjectPrefixForTest(t), string(sep)+RandomString())
```

Comment:
This shall not be picked up I think. cc @nak3

Reply:
Yes, we added this patch https://github.com/openshift/knative-serving/blob/release-next/openshift/patches/001-object.patch for upstream test and we need it.